src/main/java/com/salesforce/cdp/queryservice/util/Constants.java

```diff
@@ -55,6 +55,10 @@
     public static final String NA46_DEFAULT_CLIENT_ID = "3MVG9sA57VMGPDfeS67yma6IPflHn83FRhxVpmnuzp7R8uS42JYshQ7gWgWR63CQRgKL9gY5AfitSme.01ib6";
     public static final String NA46_DEFAULT_CLIENT_SECRET = "BDAF015C3D2418008842CAE91B0C8DD2D672B41707FB11EA3CFC5A5392E31866";
 
+    //Audience constants for different environments
+    public static final String PROD_SERVER_AUD = "login.salesforce.com";
+    public static final String TEST_SERVER_AUD = "login.test1.pc-rnd.salesforce.com";
```

Comment:
HOw about dev server aud?

Reply:
Dev server aud is same as test server aud. I have modified this constants name to - DEV_TEST_SERVER_AUD